tests/Oryx.Tests.Common/EnvironmentVariableListExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.BuildScriptGenerator.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Oryx.Tests.Common
+{
+    public static class EnvironmentVariableListExtensions
+    {
+        public static ICollection<EnvironmentVariable> AddTestStorageAccountEnvironmentVariables(this ICollection<EnvironmentVariable> envVarList)
```

Comment:
As cormac mentioned, lets add doc for this and `GetKeyVaultSecretValue`. If you aren't already familiar with this trick, you can do so easily in visual studio by making a newline right above the method and typing `///`, which will create a boilerplate doc comment. Then, sometimes github copilot will even write the doc for you :)

Reply:
Thanks for suggestion Paul. Luckily, I knew it. However, I misread Cormac's suggestion for adding docs. Will add docs accordingly.